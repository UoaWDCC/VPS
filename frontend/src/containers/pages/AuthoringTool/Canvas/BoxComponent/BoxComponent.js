import { Box, Button, makeStyles } from "@material-ui/core";
import React from "react";

const useStyles = makeStyles({
  box: {
    backgroundColor: "#008a7b",
    alignItems: "center",
    justifyContent: "center",
    display: "flex",
  },
});

export default function BoxComponent({ id, selectElement }) {
  const styles = useStyles();
  return (
    <Box
      className={styles.box}
      width={100}
      height={100}
      key={id}
      id={id}
      onClick={selectElement}
    >
      Hello World
    </Box>
  );
}
