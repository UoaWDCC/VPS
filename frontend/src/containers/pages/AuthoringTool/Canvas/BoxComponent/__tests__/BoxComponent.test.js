import React from "react";
import { render } from "@testing-library/react";
import BoxComponent from "../BoxComponent";

test("Scenario Selection page snapshot test", () => {
  const { baseElement } = render(<BoxComponent />);

  expect(baseElement).toMatchSnapshot();
});
