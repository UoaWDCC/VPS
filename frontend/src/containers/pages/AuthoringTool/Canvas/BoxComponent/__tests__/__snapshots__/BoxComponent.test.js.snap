// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scenario Selection page snapshot test 1`] = `
<body>
  <div>
    <div
      class="MuiBox-root MuiBox-root-1"
    >
      Hello World
    </div>
  </div>
</body>
`;
